sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -19,7 +19,7 @@
 if sys.version_info >= (3, 8):
     from typing import Literal  # pylint: disable=no-name-in-module, ungrouped-imports
 else:
-    from typing_extensions import Literal  # type: ignore  # pylint: disable=ungrouped-imports
+    from typing_extensions import Literal  # pylint: disable=ungrouped-imports
```

Comment:
Just import from typing_extensions all the time, there are no perf penalities for doing so, compared to the if/else here

Reply:
Updated